tests/integration/goldens/asset/samples/generated_samples/cloudasset_generated_asset_v1_asset_service_delete_feed_async.py

```diff
@@ -35,6 +35,7 @@ async def sample_delete_feed():
 
     # Initialize request argument(s)
     request = asset_v1.DeleteFeedRequest(
+        name="projects/{project}/feeds/{feed}",
```

Comment:
This syntax seems kind of weird to me - I don't understand what `{project}` and `{feed}` mean

Reply:
I'm copying the [pattern](https://github.com/googleapis/googleapis/blob/5d301f95d17d50f78e0b2a6889301c180b947357/google/cloud/asset/v1/asset_service.proto#L609) verbatim from the protos with the hope it will nudge folks towards the correct resource name.

I guess it is weird that it _almost_ looks like an f-string, and there are no `project` or `feed` variables defined. Would sometime like this be more helpful?

```py
async def sample_delete_feed():
    """Snippet for delete_feed"""
    # Create a client
    client = asset_v1.AssetServiceAsyncClient()

    # Initialize request argument(s)
    project = "my-project"
    feed = "my-feed"

    request = asset_v1.DeleteFeedRequest(
        name=f"projects/{project}/feeds/{feed}",
```

Reply:
The posted example is definitely more clear. +1 for that if it's not too much effort.

Reply:
Opened https://github.com/googleapis/gapic-generator-python/issues/965 to track making these strings nicer. Thanks for the feedback @leahecole @software-dov 😄